src/restapi/fMe.ts

```diff
@@ -3,7 +3,7 @@ import {Request, Response} from 'express';
 import {newOKCallResponseWithMarkdown} from '../utils/call_responses';
 
 export async function fMe(req: Request, res: Response): Promise<void> {
-    const message = '`access_token:` ' + req.body.context.oauth2.user.access_token;
+    const message = '`access_token:` ' + req.body.context.oauth2.user.token.access_token;
```

Comment:
we should NOT display the full value of the token, maybe the last 4 characters? (separate issue/PR)

Reply:
This was added so that an admin can get their token and store it in the config.  It's the only use case this is needed.  A better idea might be to have a `/zendesk storeConfigToken` that would:
1. get the current user access_token
2. verify they are an admin
3. store the token in the config

Reply:
@levb, @aaronrothschild, @DHaussermann what are your thoughts on fixing this for `p1`?  Is this a breaking change that prevents releasing until it is fixed?

Reply:
It is a pretty trivial fix. I'll push a PR soon regardless 👍🏼

Reply:
PR created for this issue https://github.com/mattermost/mattermost-app-zendesk/pull/52

Reply:
I've pushed a commit to this PR that changes this functionality. Previously, the `/zendesk me` command existed for the sole purpose of exposing a user's ZD `access_token` in an ephemeral post.  One user would then copy that token value and paste it in the configuration modal, for the service ZD `access_token`.  This was a one time setup for webhooks, but the `/zendesk me` command was available for all users.  @levb's concern was that the full value was on display, which was true and a bad implementation.  I have since pushed a commit to this PR that changes the way the service account access_token is saved. 

A user now only need to run the `/zendesk setup-target` command which will:
1. setup the target in Zendesk
2. save their access_token as the service account access token

Reply:
This was the PR that added the new capability.  It was merged into this PR.  https://github.com/mattermost/mattermost-app-zendesk/pull/52

Reply:
So why is that step needed though? We already have the value so why do we need the user to copy it from us, and paste it into another form (or command) to us?